src/main/java/net/cosmocat/marketplace/database/dto/request/CategoryCreateRequest.java

```diff
@@ -0,0 +1,28 @@
+package net.cosmocat.marketplace.database.dto.request;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.*;
+import lombok.Data;
+import net.cosmocat.marketplace.validation.CosmicWordCheck;
+
+import java.util.List;
+
+@Data
+@Schema(description = "Request object for creating a new category")
+public class CategoryCreateRequest {
```

Comment:
You have inconcistency - Dto, request... what is the difference? Better to stick to single terminology. Dto should it be